lib/services/downloads_service_backend.dart

```diff
@@ -1424,13 +1424,9 @@ class DownloadsSyncService {
         _downloadsService.updateItemState(canonItem, DownloadItemState.failed);
       });
     }
-    // We try to always fetch the mediaSources when getting album/playlist, but sometimes
-    // we download/sync individual songs and need to fetch playback info here.
-    BaseItemDto? extendedItem = (downloadItem.baseItem?.mediaSources != null) && (downloadItem.baseItem?.mediaStreams != null) ? downloadItem.baseItem! :
-        (await _jellyfinApiData.getItemByIdBatched(item.id,
-                "${_jellyfinApiData.defaultFields},MediaSources,MediaStreams,SortName"));
-    List<MediaSourceInfo>? mediaSources = extendedItem?.mediaSources;
-    List<MediaStream>? mediaStreams = extendedItem?.mediaStreams;
+    // At this point the baseItem should always have the needed attributes
+    List<MediaSourceInfo>? mediaSources = downloadItem.baseItem?.mediaSources;
+    List<MediaStream>? mediaStreams = downloadItem.baseItem?.mediaStreams;
```

Comment:
You shouldn't need the blocks that copy these items into the BaseItemDto now that these attributes are not being updated.